### ISSUE TITLE
Changes are not listed for users whose language is not translated by the plugin

### ISSUE THREAD

**Issue by hughbris** [hughbris/grav-plugin-pushy#35]
I have a user who is not seeing the table of changes in the Admin UI. It appeared to be only this user and they are a member of the `publishers` group. The user can see the "Publish" menu item and the adjacent badge showing the correct number of changes.

![image](https://github.com/user-attachments/assets/3748546e-f336-4319-b303-f060e206b879)

I noticed a Javascript error when I reproduced the error using a local copy of their account:

```
20:46:55.283 Uncaught (in promise) TypeError: can't access property "querySelector", allMenuItems[index] is undefined
    updateMenuBadge pushy-admin.ts:156
    fetchItems pushy-admin.ts:138
pushy-admin.ts:156:22
```
In [pushy-admin.ts](https://github.com/hughbris/grav-plugin-pushy/blob/dd27baf2c83f184e6d69077f828729097749da03/build/ts/pushy-admin.ts#L155), I can see that `index` is _-1_:

```ts
private updateMenuBadge() {
        // Find badge for Publish menuitem
        const allMenuItems = document.querySelectorAll('#admin-menu li');
        const index = Array.from(allMenuItems).findIndex(node => node.querySelector('em')?.innerHTML == pushy.translations.menuLabel);
        const badge = allMenuItems[index].querySelector('#admin-menu li a .badge.count');
```

`pushy.translations.menuLabel` is "_PLUGIN_PUSHY.MENU_LABEL_PUBLISH_"

So it's failing to find the untranslated string in the menu because it is rendered in English.

**To reproduce:** log out and change your user account language to one that the plugin doesn't feature (anything but English or Italian). Log in again and you will see the error. As an exercise, you can work around the error by adding in just this single translation into languages.yaml:

```yaml
de: # use whichever language you chose
  PLUGIN_PUSHY:
    MENU_LABEL_PUBLISH: Publisch #fictional translation!
```

**Comment by pamtbaau** [comment]
Not tested, just guessing...

Presuming `$this->grav['language']->translate()` returns the key when the lookup failed, perhaps you could add a fallback in the `translation()` function of *pushy.php* at line 476:

```
$translation = $this->grav['language']->translate(["$prefix.$key", $arg], [$language]);

if ($translation == "$prefix.$key") {
  $translation = $this->grav['language']->translate(["$prefix.$key", $arg], ['en']);
}

return $translation;
```

**Comment by hughbris** [comment]
Indeed that works and also reverts the untranslated heading to English. ! Thank you! You traced the problem further back to the PHP.

Would you like to make a PR? I'm just giving you the option because the fix is your work :)

**Comment by pamtbaau** [comment]
> You traced the problem further back to the PHP.

Yes, I'm a smart cookie... :-)

But not that smart to prevent the issue in the first place  :-(

> I'm just giving you the option because the fix is your work :)

My ego isn't that large...

Any way: [Fix method pushy.translate() #36](https://github.com/hughbris/grav-plugin-pushy/pull/36)

**Comment by hughbris** [comment]
Fixed by [PR #35](https://github.com/hughbris/grav-plugin-pushy/issues/35), thanks @pamtbaau
